controllers/numaresourcesscheduler_controller.go

```diff
@@ -109,8 +114,21 @@ func (r *NUMAResourcesSchedulerReconciler) Reconcile(ctx context.Context, req ct
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *NUMAResourcesSchedulerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// we want to initiate reconcile loop only on change under labels or spec of the object
```

Comment:
should we ignore annotations them? why or why not?

Reply:
I thought it might somehow reduce reconciliation loops, but maybe let's stay with the basics and reconcile for every change.

Reply:
I followed the controller code, there is no reason to act upon other changes because according to the code we won't revert metadata changes.
For ```Labels``` and ```Annotations``` changes, the controller will merge old and new values.
For changes under ```spec``` it will force the desired state.